content/learning-paths/microcontrollers/img_nn_stcube/build_nn.md

```diff
@@ -18,7 +18,14 @@ For each step, you will see
 
 To create the NN model, there are certain data pre-processing steps that need to be performed.
 
-Execute the first code block to import the required packages:
+First, open the Jupyter Notebook through an Anaconda Prompt.
```

Comment:
This step in the previous section under prepare, but it actually helps to reiterate in this section.